File: bubble_sort.py
data = [12, 3, 2, 6, 5, 7]


def bubble_sort(data):
    for i in range(0, len(data) - 1):
        for j in range(0, len(data) - 1 - i):
            if data[j] > data[j + 1]:
                data[j], data[j + 1] = data[j + 1], data[j]

    return data


print(bubble_sort(data))

File: bubbleSort.java
import java.util.Arrays;

public class Main {
    public static void main(String... args) {
        int[] data = {
                9, 2, 4, 1, 6
        };

        System.out.println(Arrays.toString(bubbleSort(data)));
    }

    private static int[] bubbleSort(int[] data) {
        for(int i = 0; i < data.length - 1; i++) {
            for(int j = 0; j < (data.length - 1) - i; j++) {
                if(data[j] > data[j + 1]) {
                    int v = data[j];
                    data[j] = data[j + 1];
                    data[j + 1] = v;
                }
            }
        }

        return data;
    }
}